samples/ORKCatalog/ORKCatalog/Tasks/TaskListRow.swift

```diff
@@ -1424,12 +1424,22 @@ enum TaskListRow: Int, CustomStringConvertible {
         
         let healthKitPermissionType = ORKHealthKitPermissionType(sampleTypesToWrite: healthKitTypesToWrite,
                                                                  objectTypesToRead: healthKitTypesToRead)
-        
-        let requestPermissionsStep = ORKRequestPermissionsStep(identifier: String(describing: Identifier.requestPermissionsStep),
-                                                               permissionTypes: [healthKitPermissionType])
+
+        let notificationsPermissionType = ORKNotificationPermissionType(
+            authorizationOptions: [.alert, .badge, .sound])
```

Comment:
Nit: maybe some better styling for these two lines. It's hard that this line is part of the constructor.